modules/ROOT/pages/remove-client-app-latest-task.adoc

```diff
@@ -1,12 +1,28 @@
-= Removing API instance Contracts on API Manager
+= Revoke and Delete API Contracts
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-As Organization Administrator of the master organization, to delete an application that you gave access to your API:
+As the Organization Administrator of the master organization, you can revoke or delete a contract granting an application access to your API. When you revoke a contract, you can restore the contract at any time. 
 
-. Click API Manager > Contracts.
-. Select the name of the application to delete.
-. Click Delete image:trash.png[trash icon].
+Deleting a contract deletes the approved contract from Anypoint Exchange. Before you delete a contract, you must first revoke it. 
```

Comment:
> Deleting a contract deletes the approved contract.
The user cannot delete an approved contract.

```suggestion
Revoking a contract blocks the Client Application from accessing the API. Once revoked, a contract could be restored or deleted.
```

Reply:
Done! Thanks